lib/src/models/ble_device.dart

```diff
@@ -5,19 +5,18 @@ import 'package:universal_ble/universal_ble.dart';
 class BleDevice {
   String deviceId;
   String? name;
-  bool? isPaired;
   Uint8List? manufacturerDataHead;
   Uint8List? manufacturerData;
   int? rssi;
   List<String> services;
 
   Future<bool> get isConnected => UniversalBle.isConnected(deviceId);
+  Future<bool?> get isPaired => UniversalBle.isPaired(deviceId);
```

Comment:
isPaired is not required to be a getter, its filled by NativePlatforms automatically if available